tracer/test/test-applications/integrations/Samples.CosmosDb/Program.cs

```diff
@@ -72,7 +72,14 @@ public async Task GetStartedDemoAsync()
                 { 
                     ApplicationName = "CosmosDBDotnetQuickstart", 
                     RequestTimeout = TimeSpan.FromMinutes(10),
-                    OpenTcpConnectionTimeout = TimeSpan.FromMinutes(1),
+                    // the docker container has a cert, we just ignore it, it isn't important
+                    // the docs say to do this!
+                    HttpClientFactory = () => new HttpClient(new HttpClientHandler()
```

Comment:
FWIW, if this is a problem for older versions, I'm inclined to add an `#if` here, and just don't test those old versions in CI. It's still better than what we have now anyway.